### ISSUE TITLE
Outgoing transaction with same from and to address doesnt appear in the transaction list

### ISSUE THREAD

**Issue by wouter-ub** [excelsia/vee-wallet-gui#134]
Using the web based version of the VEE wallet at https://wallet.vee.tech/

Sending any amount from an address to the same address, results in the transaction list not showing the outgoing transaction (only the incoming one).

The balance at the top of the page is correct, but the sum of the transactions in the transaction list is not.

I suggest to either don't allow people to send money to the same 'from' address, or to show both the incoming and outgoing transaction in the transaction list.

**Comment by ncying** [comment]
@wouter-ub self-payment is allowed in our project [confirmed]. do you suggest to show 'two' (income and outcome) tx separately?

**Comment by wouter-ub** [comment]
Yes, because now the sum of the transactions in the list (or the export) doesn’t match the balance of the account.

**Comment by ncying** [comment]
@wouter-ub i see
@masonExcelsia
